support/rally.js

```diff
@@ -113,8 +130,14 @@ export default class Rally {
     }
 
     switch (message.type) {
+      case "pause":
+        this._pause();
+        break;
+      case "resume":
+        this._resume();
+        break;
       case "uninstall":
-        return browser.management.uninstallSelf({showConfirmDialog: false});
+        return browser.management.uninstallSelf({ showConfirmDialog: false });
```

Comment:
Same here :)Why is your IDE unhappy with our rules? Do we need to change our rules? :)